.eslintrc.js

```diff
@@ -0,0 +1,140 @@
+module.exports = {
+  root: true,
+  overrides: [
+    {
+      files: [
+        '*.ts'
+      ],
+      plugins: [
+        'modules-newline'
+      ],
+      parserOptions: {
+        project: [
+          'tsconfig.json',
+        ],
+        createDefaultProgram: true
+      },
+      extends: [
+        'plugin:@angular-eslint/ng-cli-compat',
+        'plugin:@angular-eslint/ng-cli-compat--formatting-add-on',
+        'plugin:@angular-eslint/template/process-inline-templates',
+        'plugin:import/typescript',
+      ],
+      rules: {
+        'prefer-arrow/prefer-arrow-functions': [
+          'error',
+          {
+            allowStandaloneDeclarations: true
+          }
+        ],
+        '@typescript-eslint/naming-convention': [
+          'error',
+          {
+            selector: 'enumMember',
+            format: ['PascalCase', 'UPPER_CASE', 'camelCase']
+          }
+        ],
+        'no-shadow': 'off',
+        '@typescript-eslint/no-shadow': ['error'],
+        '@typescript-eslint/member-ordering': 'off',
+        'import/no-unassigned-import': 'warn',
+        'import/order': [
+          'error',
+          {
+            groups: [
+              ['builtin', 'external'],
+              ['internal']
+            ],
+            pathGroups: [
+              {
+                pattern: '@daffodil/**',
+                group: 'internal'
+              }
+            ],
+            pathGroupsExcludedImportTypes: ['builtin'],
+            'newlines-between': 'always',
+            alphabetize: {
+              order: 'asc',
+              caseInsensitive: false
+            }
+          }
+        ],
+        'modules-newline/import-declaration-newline': 'warn',
+        'modules-newline/export-declaration-newline': 'warn',
+        indent: [
+          'error',
+          2,
+          {
+            SwitchCase: 1
+          }
+        ],
+        'object-curly-newline': [
+          'error',
+          {
+            ImportDeclaration: {multiline: true, minProperties: 2},
+            ExportDeclaration: {multiline: true, minProperties: 2}
+          }
+        ],
+        'object-curly-spacing': [
+          'error',
+          'always',
+          {
+            arraysInObjects: false,
+            objectsInObjects: false
+          }
+        ],
+        'no-use-before-define': 'off',
+        '@typescript-eslint/no-use-before-define': [
+          'error',
+          {
+            functions: false
+          }
+        ],
+        '@angular-eslint/component-class-suffix': [
+          'error',
+          {
+            suffixes: [
+              'Component'
+            ]
+          }
+        ],
+        '@typescript-eslint/consistent-type-assertions': [
+          'error',
+          {
+            assertionStyle: 'angle-bracket'
+          }
+        ],
+        '@typescript-eslint/consistent-type-definitions': 'error',
```

Comment:
I don't quite understand this one. Isn't it supposed to be either `[ 'error', 'interface']` or `['error', 'type']`?

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/consistent-type-definitions.md

Reply:
I think the `interface` option is the default, but I can set it explicitly if you want

Reply:
for example 
```ts
type Foo = {
  thing: string
}
```
will trigger a linting error with the current setting

Reply:
changed